utils/rapids-colab.sh

```diff
@@ -0,0 +1,126 @@
+    echo "dist_package_index = sys.path.index\('/usr/local/lib/python3.6/dist-packages'\)"
+    echo "sys.path = sys.path[:dist_package_index] + ['/usr/local/lib/python3.6/site-packages'] + sys.path[dist_package_index:]"
+    echo "sys.path"
+    echo "if os.path.exists\('update_pyarrow.py'\):"
+    echo "  exec\(open\('update_pyarrow.py'\).read\(\), globals\(\)\)"
+    echo "********************************************************************************************************"
+    rm rc2.sh
+    echo "Please COPY the above code and RERUN the cell"
+    exit 0
+  fi
+else
+  echo "You may not have to change anything.  All versions of our script should work with this version of Colab"
+fi
+
+wget -nc https://github.com/rapidsai/notebooks/raw/master/utils/env-check.py
+echo "Checking for GPU type:"
+python env-check.py
+
+if [ ! -f Miniconda3-4.5.4-Linux-x86_64.sh ]; then
+    echo "Removing conflicting packages, will replace with RAPIDS compatible versions"
+    # remove existing xgboost and dask installs
+    pip uninstall -y xgboost dask distributed
+
+    # intall miniconda
+    echo "Installing conda"
+    wget https://repo.continuum.io/miniconda/Miniconda3-4.5.4-Linux-x86_64.sh
+    chmod +x Miniconda3-4.5.4-Linux-x86_64.sh
+    bash ./Miniconda3-4.5.4-Linux-x86_64.sh -b -f -p /usr/local
+    
+    if (( $RAPIDS_RESULT == 13 )) ;then #Newest nightly packages.  UPDATE EACH RELEASE!
+    echo "Installing RAPIDS $RAPIDS_VERSION packages from the nightly release channel"
+    echo "Please standby, this will take a few minutes..."
+    # install RAPIDS packages
+        conda install -y --prefix /usr/local \
+                -c rapidsai-nightly/label/xgboost -c rapidsai-nightly -c nvidia -c conda-forge \
+                python=3.6 cudatoolkit=10.0 \
+                cudf=$RAPIDS_VERSION cuml cugraph gcsfs pynvml cuspatial xgboost\
+                dask-cudf
+        # check to make sure that pyarrow is running the right version (0.15) for v0.11 or later
+        wget -nc https://github.com/rapidsai/notebooks/raw/master/utils/update_pyarrow.py
+    else #Stable packages requiring PyArrow 0.15
+        echo "Installing RAPIDS $RAPIDS_VERSION packages from the stable release channel"
+        echo "Please standby, this will take a few minutes..."
+        # install RAPIDS packages
+        conda install -y --prefix /usr/local \
+            -c rapidsai/label/xgboost -c rapidsai -c nvidia -c conda-forge \
+            python=3.6 cudatoolkit=10.0 \
+            cudf=$RAPIDS_VERSION cuml cugraph cuspatial gcsfs pynvml xgboost\
+            dask-cudf
+        # check to make sure that pyarrow is running the right version (0.15) for v0.11 or later
+        wget -nc https://github.com/rapidsai/notebooks/raw/master/utils/update_pyarrow.py
+    fi
+      
+    echo "Copying shared object files to /usr/lib"
+    # copy .so files to /usr/lib, where Colab's Python looks for libs
+    cp /usr/local/lib/libcudf.so /usr/lib/libcudf.so
```

Comment:
I believe you need to copy .so files from cuml, cugraph, xgboost, cuspatial into /usr/lib as well.